moped-editor/src/views/projects/projectsListView/ProjectsListViewExportConf.js

```diff
@@ -18,30 +23,83 @@ export const ProjectsListViewExportConf = {
   },
```

Comment:
should the project description column also have "-" for empty rows?

Reply:
+1 - it would be good to add the null handler to `project_name` and `project_description` - our DB schema does not allow these fields to be blank, but the null handler would future-proof this export.